recipes/pgenlib/meta.yaml

```diff
@@ -1,17 +1,19 @@
 {% set name = "pgenlib" %}
-{% set version = "0.91.0" %}
+{% set version = "0.92.1" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
   url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
-  sha256: 32992274214d2e6735b721544abfd0ae0c76906c4193120f7091065b88f68dfa
+  sha256: 4f3b0bf6144b2b490b9d327cb7945ac5b0870c450fc37bdbfc33417269e7121b
+  patches:
+    - 0001-setup.py.patch
```

Comment:
```suggestion
```
The c++ issue should be resolved in the new v0.92.1. @mencian, can you try to remove the patch now?